### DISCUSSION TITLE
Multi-Select in Folder view

### DISCUSSION THREAD

**Discussion by nielstron** [immich-app/immich#14307] [Feature Request]
I would like to be able to select multiple images while in the folders browser, in order to perform mass actions on these images, such as adding tags or changing locations. It would also be nice if there were also a "select all" option for selecting all images in a folder.

In my case, I am using external libraries to load in my images, which are arranged in a detailed subfolder structure that makes it easy to identify different content types, so being able to perform mass actions on images on a per folder basis would make tagging much easier.

Thank you! - by @pearcemichal

This is a re-opening of #14132, consolidating among others #12263, #14118 and parts of #9159

**Comment by kettly1260** [comment]
agree, and at the same time, I hope to add the option of selecting albums on this device or selecting all photos within the album on the mobile client, so that it can be more convenient to create corresponding albums

**Comment by dzg** [comment]
Yes, please !
I would ideally like to be able to select an actual folder, e.g., `2019` or `2001-10`, and thus select every asset inside for, say, deletion, adding to album, editing date .. etc.

**Reply by fabxyz** [reply]
@bo0tzz, this comment has 5 upvotes, we still can't select folders in the folder view.

**Comment by LukasMueller187** [comment]
I'm already glad for the folder view, as my old (big) picture collection is organized in folders which represent albums.
Being able to create albums from these folders, without CLI, would be great.

Or is there another way I'm missing?

edit: this would also be a solution [#6519](https://github.com/immich-app/immich/discussions/6519)

**Comment by arnolicious** [comment]
This was implemented in #15049 btw, for the people subscribed to this discussion :)

**Reply by nielstron** [reply]
That's awesome! Thank you so much :)
